crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -382,6 +390,157 @@ impl Persistence {
         Ok(solution)
     }
 
+    pub async fn solvable_order_after(
```

Comment:
Please, can you extend this function with a comment explaining what exactly it does? :pray: